bool[3] switch
